package.json

```diff
@@ -5,6 +5,9 @@
   "engines": {
     "node": ">=16.15"
   },
+  "workspaces": [
+    "packages/*"
+  ],
```

Comment:
Perhaps not for this PR, and definitely not caused by this PR, but the dependencies section of the top-level package.json is a bit confusing and we probably have an opportunity to tidy up the common dependencies.
Should common dependencies go top-level at all now or should they be in a common package? How would this relate to utility modules, currently in `slic-tools`?
Happy for this all to be discussed/addressed separately.

Reply:
Initially, that was the approach I wanted to go for and have all the dependencies defined at the top level. There are pros and cons to this approach however especially as stated here -https://github.com/lerna/lerna/blob/main/doc/hoist.md#disadvantages-with-hoisting. Will discuss this further.